src/iris/bin/sync_targets.py

```diff
@@ -343,20 +343,30 @@ def sync_from_oncall(config, engine, purge_old_users=True):
 
         current_name = iris_db_oncall_team_id_name_dict[oncall_id]
         new_name = oncall_response_dict_id_key[oncall_id]
+        target_id_to_rename = iris_target_name_id_dict[current_name]
         try:
             if current_name != new_name:
                 # handle edge case of teams swapping names
                 if not iris_target_name_id_dict.get(new_name, None):
-                    target_id_to_rename = iris_target_name_id_dict[current_name]
                     logger.info('Renaming team %s to %s', current_name, new_name)
                     engine.execute('''UPDATE `target` SET `name` = %s, `active` = TRUE WHERE `id` = %s''', (oncall_case_sensitive_dict[new_name], target_id_to_rename))
                 else:
-                    # there is a team swap so rename to a random name to prevent a violation of unique target name constraint
-                    new_name = str(uuid.uuid4())
-                    target_id_to_rename = iris_target_name_id_dict[current_name]
-                    name_swaps[oncall_id] = target_id_to_rename
-                    logger.info('Renaming team %s to %s', current_name, new_name)
-                    engine.execute('''UPDATE `target` SET `name` = %s, `active` = TRUE WHERE `id` = %s''', (new_name, target_id_to_rename))
+                    squatter_target_id = iris_target_name_id_dict.get(new_name)
```

Comment:
Nit: why not use [] notation instead of .get(), as we've verified above that the key is present in the dict?